third_party/proxy/meta_proxier.go

```diff
@@ -69,32 +69,21 @@ func (proxier *metaProxier) SyncLoop() {
 
 // OnServiceAdd is called whenever creation of new service object is observed.
 func (proxier *metaProxier) OnServiceAdd(service *v1.Service) {
-	if *(service.Spec.IPFamily) == v1.IPv4Protocol {
-		proxier.ipv4Proxier.OnServiceAdd(service)
-		return
-	}
+	proxier.ipv4Proxier.OnServiceAdd(service)
```

Comment:
Could you confirm this?

Reply:
I changed this to follow the meta_proxier in the upstream kubernetes repo. 
Some additional changes are needed to support dual-stack service. I will push a new commit.

Reply:
I have updated the MetaProxier related changes to support dual-stack service. Please help to review. Thanks!

Reply:
Thanks for following up the question. However, we may not want to scope supporting dual-stack into this PR because the PR is already very large and ideally a PR should focus on single purpose to reduce the complexity of review and backport. It would also be hard to realize Antrea starts to support dual-stack service in a PR named "bump k8s lib to v0.21.0". We also need to add e2e tests (or enable corresponding upstream e2e test2) when declaring Antrea supporting a new feature.
For this PR, I would suggest to keep all previous behavior (it's fine to ignore the secondary service IP as it was never supported before). And you may create another PR to support dual-stack service with proper tests added and the title highlighted.